docs/src/components/libraries_and_functions.md

```diff
@@ -0,0 +1,50 @@
+# Libraries and Functions
+
+**Valence Libraries** contain the business logic that can be applied to the funds held by **Valence accounts**. Most often, this logic is about _performing operations on tokens_, such as splitting, routing, or providing liquidity on a DEX. A **Valence library** has to be registered with a **Valence account** to be authorized to perform operations on that account's balances. **Valence Libraries** expose **Functions** that it supports. **Valence Programs** can be composed of a more or less complex graph of **Valence Accounts** and **Valence Libraries** to form a more or less sophisticated cross-chain workflow. During the course of a **Valence program**'s execution, **Functions** are called by external parties, and trigger the library's operations on the linked accounts.
```

Comment:
A Valence account has to authorize a Valence library. The valence library has no notion of the accounts that has authorized to perform operations on it.

Reply:
Fair enough .. I will re-phrase it. Valence accounts actually use the term `approve a library`.